app/preprints/-components/preprint-institutions/institution-manager/component.ts

```diff
@@ -55,29 +55,40 @@ export default class InstitutionsManagerComponent extends Component<InstitutionA
 
                 await this.manager.preprint.affiliatedInstitutions;
 
+                userInstitutions.map((institution: PreprintInstitutionModel) => {
+                    this.institutions.push(institution);
+                });
```

Comment:
Nit: couldn't this just be `this.institutions = userInstitutions;`?

Not a blocker for merge, but `.map` feels a bit strange to use here